src/DependencyInjection/Compiler/AddProviderCompilerPass.php

```diff
@@ -89,14 +89,7 @@ private function applyFormats(ContainerBuilder $container, array $config): void
                 $definition = $container->getDefinition($id);
 
                 foreach ($context['formats'] as $format => $formatConfig) {
-                    $formatConfig['quality'] = $formatConfig['quality'] ?? 80;
-                    $formatConfig['format'] = $formatConfig['format'] ?? 'jpg';
-                    $formatConfig['height'] = $formatConfig['height'] ?? null;
-                    $formatConfig['constraint'] = $formatConfig['constraint'] ?? true;
-                    $formatConfig['resizer'] = $formatConfig['resizer'] ?? false;
-
-                    $formatName = sprintf('%s_%s', $name, $format);
-                    $definition->addMethodCall('addFormat', [$formatName, $formatConfig]);
+                    $definition->addMethodCall('addFormat', [sprintf('%s_%s', $name, $format), $formatConfig]);
```

Comment:
this change is to avoid duplication on the defaults. They are already on the Configuration class. (Also I removed the false default, because the admin format does not have it).

This is also produces the rest of the diff, changes from false to null.